google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/model.scala

```diff
@@ -77,66 +77,172 @@ object PubSubConfig {
     apply(projectId, clientEmail, privateKey, pullReturnImmediately, pullMaxMessagesPerInternalBatch)(actorSystem)
 }
 
+final class PublishMessage private[pubsub] (val data: String,
```

Comment:
The constructor should be `private` as `private[pubsub]` still retains the full visibility from Java.